Orange/widgets/visualize/owpythagorastree.py

```diff
@@ -0,0 +1,416 @@
+import math
+from collections import namedtuple, defaultdict
+
+import Orange
+from Orange.widgets import gui
+from Orange.widgets.widget import OWWidget
+from Orange.widgets.settings import Setting
+from Orange.classification.tree import TreeClassifier
+from Orange.data.table import Table
+
+from PyQt4 import QtGui, QtCore
+from PyQt4.QtCore import Qt
+
+PI = math.pi
```

Comment:
You seem to define this but then later use `math.pi` as well. If you import everything like
```py
from math import pi as PI, sin, cos, radians
```
you don't need this line anymore, get shorter lines and should also be able to shave some 10ns from each math call. :smile: